test/http.cc

```diff
@@ -231,6 +234,56 @@ TEST_P(HttpTest, GetHeaders) {
   auto response = future.get();
 
   EXPECT_EQ(200, response.code());
-  EXPECT_THAT(response.headers(), testing::ContainsRegex("Foo: Bar"));
+  EXPECT_THAT(
+      response.headers(),
+      testing::Contains(http::Header("Foo", "Bar")));
+  EXPECT_EQ(response.headers().at("Content-Length"), "25");
```

Comment:
Love seeing usage of `testing::Contains()` above, let's also do that here!

Reply:
Again, just a showcase for others!